### DISCUSSION TITLE
handleChatMemberJoined method does not work

### DISCUSSION THREAD

**Discussion by Odilbukh** [def-studio/telegraph#342] [Q&A]
I have copied and pasted the code from documentation:
```
protected function handleChatMemberJoined(User $member): void
    {
        $this->chat->html("Welcome {$member->firstName()}")->send();
    }
    
```

But when new telegram user clicks to start to bot handleChatMemberJoined method does not work. Maybe it works in other command (not on /start), IDK

I want register all new users and store their data in DB.

**Comment by fabio-ivona** [comment]
Hi, where did you pasted the code?

**Reply by fabio-ivona** [reply]
@Odilbukh you should listen for `/start` command

```php
class CustomWebhookHandler extends WebhookHandler
{
    public function start()
    {
        $userFirstName = $this->message->from()->firstName();
        $this->chat->message("Welcome $userFirstName")->send();
    }
}
```
